fiftyone.yml

```diff
@@ -0,0 +1,6 @@
+fiftyone:
+  version: ~0.21.0
```

Comment:
This defines the compatible version using semver matchers.

Reply:
We probably need a way to also define the teams compatible version.

Reply:
Every version of the Teams SDK has a corresponding OSS version, so we could require all users to "speak" OSS versions for now